### PR TITLE
added ifEmpty prop that accepts custom Component rendered only if no children are passed

### PR DESCRIPTION
**Pull request by abhinavdalal-iconnect**

### CODE REVIEW

**Comment by abhinavdalal-iconnect** [comment]
this should solve this issue : 
https://github.com/YIZHUANG/react-multi-carousel/issues/72

File: README.md [hunk 1]
```diff
@@ -273,6 +274,18 @@ const CustomDot = ({ onClick, ...rest }) => {
 </Carousel>
 ```
 
+## custom ifEmpty
+You can pass your own component to be rendered only when no children are passed.
+```js
+const carouselItems = items.filter(i=> i.name.search(searchQuery))
+
+<div className='my-own-custom-container'>
+  <Carousel ifEmpty={<CustomComponent />}>
```
Comment (YIZHUANG) [inline README.md]: Is this something like a placeholder? For example showing skeleton for the incoming content.

Comment (abhinavdalal-iconnect) [reply README.md]: no this is what will be displayed when no children is passed to the carousel.. in my above example.. think the searchQuery does not match any components.. and hence carouselItems=[].. i would just like to display something like `No results found for "${searchQuery}"` with maybe a sad face svg ;).. so my ifEmpty component would be rendered when user types something in search box that is has no matches only..

Comment (abhinavdalal-iconnect) [reply README.md]: @YIZHUANG this works as you have pointed out.. closing this PR..

**Comment by abhinavdalal-iconnect** [comment]
use :
```
<Carousel>
   { hasItems ? <ItemList /> :<NotFound /> }
</Carousel>
```

**Pull request closed without merging**